haplotypecaller-gvcf-gatk4.hg38.wgs.inputs.json

```diff
@@ -10,27 +10,31 @@
 
   "##_COMMENT3": "INTERVALS",
   "HaplotypeCallerGvcf_GATK4.scattered_calling_intervals_list": "gs://gatk-test-data/intervals/hg38_wgs_scattered_calling_intervals.txt",
-  "HaplotypeCallerGvcf_GATK4.HaplotypeCaller.interval_padding": 100,
 
   "##_COMMENT4": "DOCKERS", 
   "HaplotypeCallerGvcf_GATK4.gatk_docker": "broadinstitute/gatk:4.0.0.0",
   
   "##_COMMENT5": "PATHS", 
   "HaplotypeCallerGvcf_GATK4.gatk_path": "/gatk/gatk",
+  "HaplotypeCallerGvcf_GATK4.HaplotypeCaller.make_gvcf": "True",
+  "##HaplotypeCallerGvcf_GATK4.HaplotypeCaller.contamination": "(optional) Float?",
 
   "##_COMMENT6": "JAVA OPTIONS", 
-  "HaplotypeCallerGvcf_GATK4.HaplotypeCaller.java_opt": "-Xms8000m",
-  "HaplotypeCallerGvcf_GATK4.MergeGVCFs.java_opt": "-Xms2000m",
+  "##HaplotypeCallerGvcf_GATK4.HaplotypeCaller.java_options": "(optional) String?",
 
   "##_COMMENT7": "MEMORY ALLOCATION", 
-  "HaplotypeCallerGvcf_GATK4.HaplotypeCaller.mem_size": "10 GB",
-  "HaplotypeCallerGvcf_GATK4.MergeGVCFs.mem_size": "3 GB",
+  "##HaplotypeCallerGvcf_GATK4.HaplotypeCaller.mem_gb": "(optional) Int?",
+  "##HaplotypeCallerGvcf_GATK4.MergeGVCFs.mem_gb": "(optional) Int?",
 
   "##_COMMENT8": "DISK SIZE ALLOCATION",
-  "HaplotypeCallerGvcf_GATK4.HaplotypeCaller.disk_size": 100,
-  "HaplotypeCallerGvcf_GATK4.MergeGVCFs.disk_size": 100,
+  "##HaplotypeCallerGvcf_GATK4.MergeGVCFs.disk_space_gb": "(optional) Int?",
+  "##HaplotypeCallerGvcf_GATK4.HaplotypeCaller.disk_space_gb": "(optional) Int?",
+
+  "##_COMMENT9": "CPU",
+  "##HaplotypeCallerGvcf_GATK4.MergeGVCFs.cpu": "(optional) Int?",
+  "##HaplotypeCallerGvcf_GATK4.HaplotypeCaller.cpu": "(optional) Int?",
```

Comment:
Is it worth making this an option?  I don't think either of those tasks can multi-thread.

Reply:
I can take them out. i was attempting to follow the wdl template to make all the wdls consistent but if the tools can't use the parameter maybe they shouldn't be there. https://docs.google.com/document/d/1uiapGNu4lWTmiV_xHjvDfHuDMYxTpG229gqNutQQhfQ/edit#heading=h.9kzq7s20lglw